modules/API/API.js

```diff
@@ -257,6 +263,27 @@ function onDesktopSharingEnabledChanged(enabled = false) {
     }
 }
 
+/**
+ * Sets the maximum video size the local participant should send and receive from
+ * remote participants.
+ * @param {number} frameHeight - The user preferred max frame height for send and
+ * receive video.
+ * @returns {void}
+ */
+function setVideoQuality(frameHeight) {
```

Comment:
Isn't it going to be better if we define this function somewhere else? It can be an action and we can only dispatch it from here. WDYT?

Reply:
I made the changes you suggested, can you PTAL again ? Thanks !